docs/source/datasets/index.rst

```diff
@@ -39,6 +43,9 @@ The Decennial Census dataset is a simulated enumeration of the US Census Bureau'
 that have been simulated are 2020, 2030, and 2040. To find out more about the Decennial Census, please visit the Decennial Census
 `homepage <https://www.census.gov/programs-surveys/decennial-census.html>`_.   
 
+To find out more about how to generate a simulation of the Decennial Census using Pseudopeople, please see the `API Reference page <https://pseudopeople.readthedocs.io/en/latest/api_reference/interface.html#pseudopeople.interface.generate_decennial_census>`_ for the 
+:code:`generate_decennial_census()` function. 
```

Comment:
```suggestion
To find out more about how to generate a simulation of the Decennial Census using Pseudopeople, see :func:`pseudopeople.interface.generate_decennial_census`.
```

I think we should make this short, and repeat it for every dataset. I think we need to have the "interface" part of this link for now but I'm going to follow up with the engineers about it on Slack.